.github/workflows/pynorms.yaml

```diff
@@ -0,0 +1,24 @@
+name: Check Norms
+on: [push, pull_request]
```

Comment:
Should we add `workflow_dispatch` so we can run it manually in GitHub? This didn't make sense for the differential shellcheck, since it requires a PR to diff, but here it would be sensible.

Reply:
Not sure how a manual dispatch would produce a different result from a push or pull_request, as a commit needs to be made to fix any errors.
But sure.

Reply:
Deferred to a future PR.